cgmanifest.json

```diff
@@ -22440,6 +22440,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "rpm-mpi-hooks",
+          "version": "6",
+          "downloadUrl": "https://src.fedoraproject.org/rpms/rpm-mpi-hooks/blob/rawhide/f/mpi.attr"
```

Comment:
I forgot to mention, that we also need to update the spec files themselves, so they contain the correct URLs. Passing the GitHub checks for these specs is a good indicator, if we've done it right or not.